e2e/app/page/cohort-participant-detail-page.ts

```diff
@@ -20,8 +21,46 @@ export default class CohortParticipantDetailPage extends AuthenticatedPage {
     return true;
   }
 
+  // get the particpant ID displayed between the two pi-angle button on participant detail page
+  async getParticipantIDnum(): Promise<string> {
+    const selector = `//*[@class="detail-header"]//span[contains(., "Participant ")]`;
+    const pID = await this.extractParticipantID(selector);
+    console.log(`Participant ID: ${pID}`);
+    return pID;
+  }
+
+  // extract only the number of the participant
+  async extractParticipantID(selector: string): Promise<string> {
+    const elemt = await this.page.waitForXPath(selector, {visible: true});
+    const textContent = await getPropValue<string>(elemt, 'textContent');
+    const regex = new RegExp(/\d+/);
+    return (regex.exec(textContent))[0];
+  }
+  
+  // get the back to review set button
   async getBackToReviewSetButton(): Promise<Button> {
     return Button.findByName(this.page, {name: LinkText.BackToReviewSet});
   }
+ 
+   // click on the pen icon located on the side bar
+   async clickPenIconHelpSideBar(): Promise<void> {
+    const clickPenIcon = await this.page.waitForXPath(`//*[@data-test-id="help-sidebar-icon-annotations"]`);
+    await clickPenIcon.click();
+    await this.page.waitForXPath('//*[@data-test-id="sidebar-content"]', {visible: true});
+    await waitWhileLoading(this.page);
+  }
+
+  // click on pi-angle-right button to go to next participant
+  async getGoToTheNextParticipant(): Promise<void> {
+     const iconXpath = '//button[@title ="Go To the Next Participant"]';
+    await this.page.waitForXPath(iconXpath, {visible: true}).then(icon => icon.click());
+  }
```

Comment:
click is happening inside function, function name `get` is not correct. same in next function.

Reply:
ok! So do you mean I should change the function name to clickGoToNextParticipant?

Reply:
for me, I think `GoToNextParticipant` is good

Reply:
sounds good! Thanks!

Reply:
done!